packages/x-tree-view/src/internals/plugins/useTreeViewNodes/useTreeViewNodes.test.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { createRenderer, ErrorBoundary } from '@mui-internal/test-utils';
+import { RichTreeView } from '@mui/x-tree-view/RichTreeView';
+import { SimpleTreeView } from '@mui/x-tree-view/SimpleTreeView';
+import { TreeItem } from '@mui/x-tree-view/TreeItem';
+
+describe('useTreeViewNodes', () => {
+  const { render } = createRenderer();
+
+  it('should throw an error when two items have the same ID (items prop approach)', function test() {
+    // TODO is this fixed?
```

Comment:
The grid could not run similar tests with karma, I tried a little but did not want to invest to much time on it